### DISCUSSION TITLE
【求助】有没有好用的音乐软件

### DISCUSSION THREAD

**Discussion by keerdun** [share121/inter-knot#4638] [灌水]
![1725695888641](https://github.com/user-attachments/assets/f32176f0-0989-4310-800e-0ae583373cdf)
现在用的音乐软件要不就是没版权，要不就是广告多，好多都要VIP，求一个好用的
赏金500丁尼

**Comment by APHKON** [comment]
建议用洛雪音乐，导入外部的音乐源就行了

**Reply by sensei-is-playing-GenShin** [reply]
版权问题可以用酷我音乐，但很多歌都要VIP才能听和下载，但可以下载PC端或手机端的破解版，送上一个链接应该是可以下载的：https://www.dadighost.com/soft/73687.html

**Comment by 1luik** [comment]
汽水
